datalad/metadata/aggregate.py

```diff
@@ -227,6 +227,8 @@ def _extract_metadata(agginto_ds, aggfrom_ds, db, merge_native, to_save):
             continue
         # only write to disk if there is something
         objrelpath = _get_obj_location(objid, label)
+        if store is json_py.dump2xzstream:
```

Comment:
feels like we might want to have the choice as a config option eventually,  but I guess for now should suffice.

Reply:
if only there was a test though which would help to spot whenever regression might happen